packages/components/src/dropdown/index.js

```diff
@@ -96,7 +96,9 @@ export default function Dropdown( {
 					headerTitle={ headerTitle }
 					focusOnMount={ focusOnMount }
 					{ ...popoverProps }
-					anchorRef={ containerRef.current }
+					anchorRef={
+						popoverProps?.anchorRef ?? containerRef.current
```

Comment:
It would be useful to get a plus one on if this is a reasonable change.

Reply:
It looks good to me! Component becomes more flexible so 👍

Reply:
I was thinking about how we should pass the ref though. I think it would be better to pass the whole ref, rather then the value of it. So instead of `popoverProps: { anchorRef: ref.current }` we should do `popoverProps: { anchorRef: ref }`, but I'm not sure if it makes any difference.

Reply:
This looks fine to me too. 👍

Reply:
Yeah, I get what you mean. Unfortunately, we need to keep it as is because we don’t want to change the shape of the Popover component’s API, which does expect the value of the ref. (I.e. it expects what we pass to be an HTML element)

Dropdown is just transparently passing those props through to Popover, so I don’t think we should change the props here